lightning/src/ln/channelmanager.rs

```diff
@@ -3468,52 +3477,6 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 		}
 	}
 
-	/// Updates channel state based on a disconnected block.
-	///
-	/// If necessary, the channel may be force-closed without letting the counterparty participate
-	/// in the shutdown.
-	pub fn block_disconnected(&self, header: &BlockHeader) {
```

Comment:
Commit 6b1eb82e510a9bfb2366307fe83536e5deb8f5a9 says "Drop ChannelManager::block_connected() entirely" but it looks like `block_disconnected` is dropped.